drivers/performance_measurement/app/models/performance_measurement/result_calculation.rb

```diff
@@ -498,6 +498,71 @@ def time_to_move_in_median(detail, project: nil)
       )
     end
 
+    # FIXME
```

Comment:
Add a one-liner on what is unaddressed here?

Reply:
These were to remind me where I needed to update code, which I did, but they didn't.